src/main/java/uk/gov/justice/hmpps/prison/aop/connectionproxy/OracleConnectionAspect.kt

```diff
@@ -56,7 +56,7 @@ class OracleConnectionAspect(
 
   @Throws(SQLException::class)
   private fun Connection.openProxySessionForCurrentUsername(): OracleConnection {
-    val currentUsername = authenticationFacade.currentPrincipal
+    val currentUsername = authenticationFacade.principal
```

Comment:
by the time we get here we've already discovered it is a nomis so no danger of the authentication not being set, in fact better to thrown an exception if it isn't set.